### PR TITLE
chore: fix for vault race condition.

### PR DESCRIPTION
**Pull request by Hamza442**

### CODE REVIEW

File: dataeng/resources/remote-config.sh [hunk 1]
```diff
@@ -92,12 +92,31 @@ unassume_role
 #     }
 # }
 
+# set up token path
+export VAULT_TOKEN_PATH=${WORKSPACE}/vault-config/vault-token
+# set path for config file which store token in each job workspace
+export VAULT_CONFIG_PATH=${WORKSPACE}/vault-config/vault_config
```
Comment (AbdouSeck) [inline dataeng/resources/remote-config.sh:98]: This config file is where the `token_helper` config is defined? How is the config file generated? I am not seeing it tracked in the git history.

**Review by AbdouSeck — CHANGES_REQUESTED** [review]

File: dataeng/resources/remote-config.sh (hunk 1 above)
Comment (Hamza442) [reply dataeng/resources/remote-config.sh:98]: This is generate from vault-config.sh file

Comment (Hamza442) [reply dataeng/resources/remote-config.sh:98]: https://github.com/edx/jenkins-job-dsl/blob/master/dataeng/resources/vault-config.sh

File: dataeng/resources/token-helper.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```
Comment (AbdouSeck) [inline dataeng/resources/token-helper.py:1]: This file needs to be executable. `chmod +x token-helper.py` should take care of that.

**Review by AbdouSeck — CHANGES_REQUESTED** [review]

**Review by adzuci — APPROVED** [review]
LGTM

File: dataeng/resources/token-helper.py (hunk 2 above)
Comment (Hamza442) [reply dataeng/resources/token-helper.py:1]: Earlier we didn't made this executable still it was getting executed referring to this one http://jenkins.analytics.edx.org/job/analytics-exporter-worker/17946/console I am wondering do we need this in this case

**Pull request merged**